src/org/etools/j1939_84/controllers/DataRepository.java

```diff
@@ -102,4 +103,33 @@ public long getPart11StartTime() {
     public void setPart11StartTime(long part11StartTime) {
         this.part11StartTime = part11StartTime;
     }
+
+    public Collection<OBDModuleInformation> getOBDModules(Integer moduleAddress) {
```

Comment:
These methods should have unit tests.

Reply:
In eclipse they pass.  In ant they fail (I'll investigate).  Because these methods are not new, only moved, they should already be covered by tests.  This was a automated eclipse refactoring, with the addition of the hastable.